docs/containers.conf.5.md

```diff
@@ -971,8 +971,14 @@ On Mac, the default volumes are:
 **provider**=""
 
 Virtualization provider to be used for running a podman-machine VM. Empty value
-is interpreted as the default provider for the current host OS. On Linux/Mac
-default is `QEMU` and on Windows it is `WSL`.
+is interpreted as the default provider for the current host OS.
+
+| Platform | Default                          | Optional |
+| -------- | ---------------------------------| -------- |
+| Linux    | QEMU                             | None     |
+| Windows  | WSL: Windows Subsystem for Linux | HyperV: Windows Server Virtualization |
+| Mac      | vfkit: Apple Hypervisor          | krunkit: Launch machine via libkrun platform, optimized for sharing GPU with the machine  |
```

Comment:
it is not called vfkit, it must use `applehv`
same for krunkit, you must use `libkrun` as provider name